src/main/java/com/kylenicholls/stash/parameterizedbuilds/eventHandlers/PushHandler.java

```diff
@@ -66,4 +68,35 @@ public void onEnd(ChangeSummary summary) throws IOException {
             return false;
         }
     }
+
+    boolean validateCommitMsg(Job job) {
+        String ignoreCommitMsg = job.getIgnoreCommitMsg();
+        if(ignoreCommitMsg.isEmpty()){
+            return true;
+        }
+        else{
+            CommitRequest commitRequest = new CommitRequest.Builder(
+                    repository, refChange.getToHash()).build();
+            String actualCommitMsg = commitService.getCommit(commitRequest).getMessage();
+            boolean hasIgnoreMsg = actualCommitMsg != null &&
+                    actualCommitMsg.toLowerCase().contains(ignoreCommitMsg.toLowerCase());
+            return !hasIgnoreMsg;
+        }
+    }
+
+    boolean validateComitter(Job job){
+        String ignoreComitters = job.getIgnoreComitters();
+        if(ignoreComitters.isEmpty()){
+            return true;
+        }
+        else{
+            String[] ignoreComitterList = job.getIgnoreComitters().toLowerCase().split(",");
```

Comment:
Unfortunately, Bitbucket usernames can have commas in them. In fact, usernames can contain most characters but it looks like newlines don't work. I'd recommend splitting on newlines instead.